### PR TITLE
Add support of Gandi SSL certificate

### PR DESCRIPTION
**Pull request by Ducatel**
Related to prometheus/prometheus#911

### CODE REVIEW

**Comment by matthiasr** [comment]
We really shouldn't go back to the fat Ubuntu-based images just for this one certificate. There must be a more subtle way of getting it in? Where doe Alpine get its certificates?

**Comment by discordianfish** [comment]
Yes, let's not go back to the ubuntu base images. Maybe it's enough to run apk-install ca-certificates?

**Comment by Ducatel** [comment]
yes, you must add `apk add ca-certificates`, but it doesn't solve the problem with Gandi registar.

But we cannot install Gandi (or other) CA file:
```
apk update && apk install curl ca-certificates \
&& curl https://www.gandi.net/static/CAs/GandiStandardSSLCA2.pem -o /usr/local/share/ca-certificates/GandiStandardSSLCA2.crt \
&& update-ca-certificates
/#  WARNING: ca-cert-GandiStandardSSLCA2.pem does not contain exactly one certificate or CRL: skipping
```
It exist an issue about this problem --> https://github.com/gliderlabs/docker-alpine/issues/30

So I have change to ubuntu for that.

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,20 @@
-FROM        sdurrheimer/alpine-glibc
-MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
+FROM ubuntu:14.04
+MAINTAINER D.Ducatel
```
Comment (juliusv) [inline Dockerfile]: This line should not be changed, independent of how the SSL cert problem is going to be fixed. Any Dockerfiles under github.com/prometheus/... should list `The Prometheus Authors <prometheus-developers@googlegroups.com>` as maintainers. The `AUTHORS.md` file is the right place to put info about individuals.

**Comment by juliusv** [comment]
Ok, let's see what happens on https://github.com/gliderlabs/docker-alpine/issues/30. We definitely do not want to change back to Ubuntu because that will blow up the Docker image size by an order of magnitude again, sorry!

**Comment by Ducatel** [comment]
OK, so waiting for update on docker-alpine

**Pull request closed without merging**